etl_project/assets/extract_news.py

```diff
@@ -174,3 +173,43 @@ def download_from_s3(
     return df
 
 
+
+def calculate_word_frequency(article_contents, word):
+    """Calculates number of times a word appears in the article"""
+    return article_contents.lower().count(str(word).lower())
+
+def process_articles(word_by_grade_level_data, newspaper_articles_data):
+    """Processs article and returns a data frame that has article title and word frequency"""
+    word_by_grade_level_df = pd.DataFrame(word_by_grade_level_data)
```

Comment:
I'm assuming, but if word_by_grade_level_data is a csv file, then maybe we're looking for pd.read_csv() instead of pd.DataFrame()